### DISCUSSION TITLE
[Protokoll] 27.02.2024

### DISCUSSION THREAD

**Discussion by 7ben18** [AdversarialAttacks/.github#13] [Protokolle]
### Wer hat an dieser Sitzung teilgenommen?

- [X] Gabriel Torres Gamez (geleitet) 
- [X] Si Ben Tran
- [X] Daniel Perruchoud
- [X] Stephan Heule

### Welche sind die wichtigsten Themen, die an dieser Sitzung besprochen wurden?

An der Sitzung wurden die 2 gesammelten Ideen und der aktuelle Stand der Projektvereinbarung gezeigt. 
Für die Notizen dazu, siehe Kommentar unten.

### Wo haben wir Hindernisse? Was blockiert den Fortschritt unser Projekts?

keine

### Was wurde an der Sitzung beschlossen?

- Wir werden uns für die nächste Woche vornehmen, dass wir weiter nach potenziellen Ideen suchen um einen grossen Pool an Möglichkeiten zu erhalten und ein grösseres Domänenwissen aufzubauen. ChatGPT sollten wir für neue Themen konsultieren. 
- Wir werden uns auch bei Monika Schlatter melden, um die Guidelines in Bezug auf KI Tools in der BAT einzuholen. 
- Die Projektvereinbarung müssen wir wenn möglich kürzen und nur die relevantesten Informationen beibehalten und das Feedback (siehe Kommentar unten) umsetzen.

**Comment by gabrieltorresgamez** [comment]
Notizen:

Projektvereinbarung
Sitzung generell Vorort, solange sich niemand meldet.
Chronologische Reihenfolge von Pflichtterminen.
Monika Schlatter - Aktuelle Guidelines der Bachelorarbeit einholen in Bezug auf ChatGPT und Copilot.
I4DS Slurm
Risiken: Dani macht sich nochmals Gedanken zu den Risiken
Zeitmanagement ist kein Risiko! (Alle haben das Risiko)
Risiken weiter machen
Projektvereinbarung: Viele Einheiten: 7 und 8, 9 und 10 -> Tendenziell zu viele Unterabschnitte, einwenig vereinfachen.
Wissenschaftliches Vorgehen: Wo stehen Methoden? Art und Methoden.
Stephan: Warum brauchen wir 9 Seiten? -> Das was wir daruber reden.
FHNW will projektvereinbarung in 2 Seiten. Ausgangslage, Problem und wie gehen wir diese an? Es muss klar werden, dass die Projektvereinbarung die Basis ist.
Wenige wichtige Sachen Zusammenfassen.
Projektvereinbarung: Kapitel zusammenbringen und der Kernaspekt sollte ich zwei Seiten möglich sein!
Idee 1
Use Case: Müssen schauen im Rahmen der Projektvereinbarung, offen genug formuliert ist.

Fragestellung High Level genug haben und Optionen einfügen in der Projektvereinbarung und diese Beschreiben!

Interessante Idee

Nicht nur Image Caption Modelle bei Black Box

LLM und ChatGPT koennen Helfen Luecken in der Forschung Entdecken als Ideengeber nutzen.

udotcom? existiert das schon? Moderne Tools als Hilfestellung nutzen.

Meinung der Betreuer: Dani muss sich das noch ueberlegen. Kann noch keine Auskunft geben.

Feuer und Flamme dann Go for It!

Fächer aufpalten, Strategien überlegen, was passt und was intressiert uns am mesitne

Hinterkopf: Erste Attacke: Eigenbetrieb muss Erkennbar sein! Unseren Beitrag muss erkennbar sein! Abheben auf das was schon gemacht wurde! Klar werden, dass es genug Fleisch am Knochen hat! Breit genug verfasste Projektvereinbarung.

Leitfragen von der Arbeit Offenformulieren und Zwischenkapitel generisch halten aber jedoch spezifisch für eine Richtung.

Netzwerk Robust machen: Geometrische Objekte

Innerhalb des Geometrischen Objektes ist keine Attacke möglich. (Sonotop im Bild)

Idee 2
Reeinforcment Learning Framgework würde hineinpassen
Sehr Hochdimensionale Entscheidung
Muss mit Frustration umgehen koennen
Admin
I4DS ist Auftraggeber

GitHub Username Stephan eingeladen

GitHub Username Dani -> Gibt uns noch bescheid

Wiederkehrender Termin sofort schicken!
